### PR TITLE
py-awscli2: update to version 2.7.10

### PR DESCRIPTION
**Pull request by steenzout**
#### Description

* update py-awscli2 to version 2.7.10

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.4
Xcode 13.4.1

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-awscli2/Portfile [hunk 1]
```diff
@@ -19,9 +19,9 @@ long_description    {*}${description}
...
 
-checksums           rmd160  567ed0aa00e8f9d820df304d09782f4d7ad79daa \
-                    sha256  d1a85bbcfa408b7e304172ff1ad207924e5d86985a6e43f82af4065a6cd47dae \
-                    size    11164419
+checksums           rmd160  3dfa678ef5b3ee86f041b7aefcb150f4839bfe71 \
+                    sha256  86dddfb0fada8b7f73c62c5f09f3e0dcaf5ddf665b5dc1de300738cb7fcd3941 \
+                    size    11188537
 
 # This is what Amazon currently supports and there
 # are hard errors with later Pythons right now.
```
Comment (steenzout) [inline python/py-awscli2/Portfile:27]: @dgilman is this still true?

```
Requirements

The aws-cli package works on Python versions:

    3.7.x and greater
    3.8.x and greater
    3.9.x and greater
    3.10.x and greater
```

from https://github.com/aws/aws-cli.

Comment (dgilman) [reply python/py-awscli2/Portfile:27]: It is still true. You are looking at the master branch, the aws-cli v2 is on the v2 branch.

**Comment by dgilman** [comment]
LGTM

File: python/py-awscli2/Portfile (hunk 1 above)
Comment (steenzout) [reply python/py-awscli2/Portfile:27]: got it. https://github.com/aws/aws-cli/tree/v2 thank you.

**Pull request merged**